external/fv3fit/fv3fit/pytorch/system.py

```diff
@@ -1,3 +1,14 @@
+import torch.backends
 import torch
+import os
 
-DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+if os.environ.get("TORCH_CPU_ONLY", False):
```

Comment:
Any plans for documenting this env var?

Reply:
Presently no, still too much in flux. I added this only because on the MPS backend it sometimes segfaults instead of giving helpful exceptions. It should only affect myself for now, since nobody else has an M1.